core/app/controllers/uffizzi_core/api/cli/v1/accounts/projects_controller.rb

```diff
@@ -17,7 +17,7 @@ class UffizziCore::Api::Cli::V1::Accounts::ProjectsController < UffizziCore::Api
 
   def create
     project_form = UffizziCore::Api::Cli::V1::Project::CreateForm.new(project_params)
-    project_form.account = current_user.organizational_account
+    project_form.account = current_user.personal_account
```

Comment:
Let's move the `default_account` definition to the app and use it in both core and the platform